backend/src/main/java/com/bakdata/conquery/models/query/resultinfo/printers/JsonResultPrinters.java

```diff
@@ -3,24 +3,25 @@
 import java.util.List;
 
 import com.bakdata.conquery.models.query.PrintSettings;
-import com.bakdata.conquery.models.query.resultinfo.printers.common.ToStringPrinter;
+import com.bakdata.conquery.models.query.resultinfo.printers.common.DateRangeStringPrinter;
+import com.bakdata.conquery.models.query.resultinfo.printers.common.DateStringPrinter;
 import lombok.ToString;
 
 /**
- * This class simply put's out native types where possible to let Jackson handle the Serialization, except for Date and DateRange, where the Frontend cannot ensure proper handling.
+ * This class simply put's out native types where possible to let Jackson handle the Serialization.
+ * Except for Date and DateRange, where the Frontend cannot ensure proper handling.
  */
 @ToString
 public class JsonResultPrinters extends JavaResultPrinters {
 
-
 	@Override
 	public Printer<Number> getDatePrinter(PrintSettings printSettings) {
-		return new ToStringPrinter<>(super.getDatePrinter(printSettings));
+		return new DateStringPrinter(printSettings);
 	}
 
 	@Override
 	public Printer<List<Integer>> getDateRangePrinter(PrintSettings printSettings) {
-		return new ToStringPrinter<>(super.getDateRangePrinter(printSettings));
+		return new DateRangeStringPrinter(printSettings, StringResultPrinters.NEGATIVE_INF_DEFAULT, StringResultPrinters.POSITIVE_INF_DEFAULT);
```

Comment:
Sollten die Konstanten nicht besser in PrintSettings sein?
Dort kann sich der DateRangeStringPrinter direkt draus bedienen.